custom_components/gismeteo/translations/sensor.pl.json

```diff
@@ -0,0 +1,21 @@
+{
+  "state": {
+    "gismeteo__condition": {
+      "clear-night": "Pogodna noc",
+      "cloudy": "Pochmurno",
+      "exceptional": "Wyjątkowa pogoda",
```

Comment:
Please, replace it to «Warunki nadzwyczajne» for consistency with weather object translation.

Reply:
Yes, "Warunki nadzwyczajne" should be used, I've edited patch-2 (but I'm still new to github, so I'm not sure if I did it properly).

I've found your "car wash" and "snowtire" integrations, they are so interesting, so I've checked Gismeteo service - it has also polish version https://www.gismeteo.pl/

Maybe your integrations will be more popular in Poland:
http://forum.jdtech.pl/Watek-hacs-home-assistant-community-store-alternatywny-sklep-z-dodatkami-dla-ha?pid=49523#pid49523
http://forum.jdtech.pl/Watek-hacs-home-assistant-community-store-alternatywny-sklep-z-dodatkami-dla-ha?pid=50069#pid50069
https://forum.arturhome.pl/t/prognoza-pogody-alternatywy-dla-preinstalowanego-met-no/2083
https://forum.arturhome.pl/t/integracje-okolopogodowe/2085